templates/base.html

```diff
@@ -7,7 +7,9 @@
 <link rel=stylesheet href=/static/css/edit.css>
 <script src=/static/js/utils.js></script>
 
-<h1>Cafebabel</h1>
+<a href=# class=logo>
```

Comment:
`href={{ url_for('home') }}` will insert the url for the homepage (probably `/`).

Reply:
yes !